recipes/espnet-tts-frontend/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Keith Ito
```

Comment:
Switch from the PYPI to the GitHub source so that this license file comes from the source tar instead of being manually included here.

https://github.com/espnet/espnet_tts_frontend/archive/refs/tags/v0.0.3.tar.gz

Reply:
Fixed it.